src/main/java/nl/novi/techiteasy/services/TelevisionServiceImpl.java

```diff
@@ -44,15 +93,17 @@ public TelevisionDto saveTelevision(TelevisionInputDto television) {
 
         return savedTv;
     }
+
     @Override
     public TelevisionDto updateTelevision(TelevisionInputDto updateTelevision, Long id) {
         if (idPresent(id)) {
-           Television tv = saveChanges(id, updateTelevision);
+            Television tv = saveChanges(id, updateTelevision);
```

Comment:
Dit is een mooie werkwijze om je code uit elkaar te trekken in verschillende methodes met alleen hun eigen nut.